src/clusterfuzz/_internal/cron/cleanup.py

```diff
@@ -1079,6 +1079,20 @@ def update_component_labels(policy, testcase, issue):
   issue.save(new_comment=issue_comment, notify=True)
 
 
+def _sanitize_ccs_list(ccs_list):
+  """Remove and log all entries with trailing comments.
+
+  Eg: Do not add "xyz@test.com #{LAST_RESORT_SUGGESTION}".
+  """
+  ret_list = []
+  for cc in ccs_list:
+    if len(cc.split()) == 1:
+      ret_list.append(cc)
+    else:
+      logs.log(f'google_issue_tracker: Filtering out CC "{cc}"')
```

Comment:
Can you change this to `{cc!r}` from `"{cc}"`

Reply:
Oops I completely missed this comment. Will piggyback this change into a future PR.